arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -0,0 +1,1670 @@
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers": {
+      "get": {
+        "tags": [
+          "Managers"
+        ],
+        "description": "Retrieves all the managers in a resource group.",
+        "operationId": "Managers_ListByResourceGroup",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The collection of StorSimple managers.",
+            "schema": {
+              "$ref": "#/definitions/ManagerList"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers/{managerName}": {
+      "get": {
+        "tags": [
+          "Managers"
+        ],
+        "description": "Returns the properties of the specified manager name.",
+        "operationId": "Managers_Get",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The StorSimple manager.",
+            "schema": {
+              "$ref": "#/definitions/Manager"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "Managers"
+        ],
+        "description": "Creates or updates the manager.",
+        "operationId": "Managers_CreateOrUpdate",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "Manager",
+            "in": "body",
+            "description": "The manager.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Manager"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/managerNameParameter"
```

Comment:
In regards to linter failure M2063, the linter is complaining about the name of this parameter (as you noted it should be named "parameters").  Please see https://github.com/Azure/azure-rest-api-specs/blob/master/arm-redis/2016-04-01/swagger/redis.json#L93 for an example of what this should look like.